lib/mix/tasks/swagger.generate.ex

```diff
@@ -188,6 +189,13 @@ defmodule Mix.Tasks.Phx.Swagger.Generate do
         swagger_map # host / port may be {:system, "ENV_VAR"} tuples or loaded in Endpoint.init callback
       end
 
+    swagger_map =
+      if scheme == "https" do
+        Map.put_new(swagger_map, :schemes, ["https", "http"])
```

Comment:
Is plain `http` included for local development, even if the production endpoint is always `https` ?